scripts/release-3-publish.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+VERSION=$1
+: "${VERSION:?'The version string is needed as parameter 1.'}"
+
+: "${GITHUB_TOKEN:?'The GITHUB_TOKEN environment variable must be defined and have repo write permissions.'}"
+
+# assume a default user of "natcap" if none has been provided.
+GH_USER="${2:-'natcap'}"
+if [ "$GH_USER" = "natcap" ]
+then
+    BUCKET="$(make jprint-RELEASES_BUCKET)/invest"
+else
+    BUCKET="$(make jprint-DEV_BUILD_BUCKET)/invest/$GH_USER"
+fi
+
+set -e
+
+source ./ci/release/RELEASE_MANAGER.env
+
+./scripts/check_required_programs.sh pandoc twine gsutil hub envsubst
+
+make dist build
+gsutil cp "$BUCKET/$VERSION/*.zip" dist  # UG, sampledata, mac binaries
+gsutil cp "$BUCKET/$VERSION/*.exe" dist  # Windows installer
+gsutil cp "$BUCKET/$VERSION/natcap.invest*" dist  # Grab python distributions
+
+RELEASE_MESSAGE_FILE=build/release_message.md
+./ci/release/build-release-text-from-history.sh > $RELEASE_MESSAGE_FILE
+hub release create \
+    --file $RELEASE_MESSAGE_FILE \
+    --committish "$VERSION" \
+    "$VERSION"
+
+PR_MSG_TEXT=build/pr_msg_text_$VERSION.txt
+SOURCE_BRANCH="master"  # TODO: does this need to be exported?
+BUGFIX_VERSION="$VERSION"  # TODO: does this need to be exported?
```

Comment:
Noticed a couple TODO comments here. Are these still active questions?

Reply:
Ah, thanks for the reminder about this!  I had forgotten to implement the fix.  The best answer here, I think, is not to `export` the variables but instead provide them directly to the subshell that runs `envsubst` where they're needed.  That way we don't unnecessarily pollute the parent shell with extraneous environment variables.  Kind of a neat feature of bash-like shells!  Patched in 4e6d4cdf